lib/transforms/type_legalizer.cc

```diff
@@ -1064,6 +1064,7 @@ bool TypeLegalizer::RunOnBasicBlock(BasicBlock* bb) {
   bool changed = false;
   for (auto& it : *bb) {
     Instruction* inst = it.get();
+    RunOnInstruction(inst);
```

Comment:
Are we missing some handling for some instructions?
In line 1071-1080, there is a switch. We should handle it there.
If it's not handled, something else is wrong.